valohai/internals/yaml.py

```diff
@@ -121,3 +118,27 @@ def get_parameter_type_name(name: str, value: Any) -> str:
         "Unrecognized parameter type for %s=%s. Supported Python types are float, int, string and bool."
         % (name, value)
     )
+
+
+def get_command(relative_source_path: str) -> List[str]:
+    if is_notebook_path(relative_source_path):
+        return get_notebook_command(relative_source_path)
+
+    return [
+        "pip install -r requirements.txt",
+        "python %s {parameters}" % relative_source_path,
+    ]
+
+
+def get_source_code(source_path: str) -> str:
+    with open(source_path) as source_file:
+        file_contents = source_file.read()
+        if is_notebook_path(source_path):
+            notebook_content = parse_ipynb(file_contents)
+            return get_notebook_source_code(notebook_content)
+        return file_contents
+
+
+def is_notebook_path(source_path: str) -> bool:
+    filename, extension = os.path.splitext(source_path)
```

Comment:
it's a good practice to prefix essential but unused defined variables with an underscore to remove IDE error highlighting for better cross-dev communication

not a must fix for now but here it would be:

```python
_filename, extension = os.path.splitext(source_path)
```

or just:

```python
_, extension = os.path.splitext(source_path)
```